src/api/helpers/nftHelpers.ts

```diff
@@ -52,7 +53,7 @@ export async function populateStat(
   try {
     const marketplaceId = query.filter?.marketplaceId;
     const owner = query.filter?.owner;
-    const stat = await NFTService.getStatNFT(NFT.serieId, marketplaceId, owner)
+    const stat = await NFTService.getStatNFT(NFT.serieId, marketplaceId, owner, query)
```

Comment:
If we need to add the "query" parameter to getStatNFT, might as well just pass only the series and the query and not marketplaceId and owner